rating_curves/build_rating_curve.R

```diff
@@ -54,14 +58,8 @@ CheckLoad <- function(path, filename) {
     loaded.data <- readRDS(paste0(path, "/", filename))
     
   } else {
-    logger::log_error("{filename} not found!")
-    logger::log_error(
-      "Run rock_stage_webscrape.R to build Pineview stage record."
-    )
-    logger::log_error(
-      "Run get_FoCo_OpenData_flow.R to build tidy Poudre Park flow record."
-    )
-    stop()
+    logger::log_warn("{filename} not found!")
+    loaded.data = F
```

Comment:
This error catch was simplified. Now, if a file cannot be found we simply return a False value from the function, which indicates that we should try to create the missing dataset.